.github/workflows/binary-applications.yml

```diff
@@ -93,33 +93,22 @@ jobs:
               # Secrets not available in PR so don't use GCP.
               if: github.event_name != 'pull_request'
               shell: bash -l {0}
+              env:
+                CERT_FILE: Stanford-natcap-code-signing-cert-expires-2024-01-26.p12
+                CERT_PASS: ${{ secrets.WINDOWS_CODESIGN_CERT_PASS }}
               run: |
                 # figure out the path to signtool.exe (it keeps changing with SDK updates)
                 SIGNTOOL_PATH=$(find 'C:\\Program Files (x86)\\Windows Kits\\10' -type f -name 'signtool.exe*' | head -n 1)
                 INSTALLER_BINARY=$(find "$(pwd)/dist" -type f -name 'InVEST_*.exe' | head -n 1)
 
-                # Specify which python version we want to use (it's the one we're using
-                # in actions/setup-python.
-                export CLOUDSDK_PYTHON=$(which python)
-                export CLOUDSDK_GSUTIL_PYTHON=$(which python)

```

Comment:
Pretty sure this has no effect - the default python should be the result of `which python`